### ISSUE TITLE
Invalid dims failure in 32-bit Python on Windows

### ISSUE THREAD

**Issue by matthew-brett** [nipy/dipy#1472]
I'm just building Dipy wheels for Windows, for Dipy 0.13.0

32 and 64 bit Pythons 2.7, 3.4, 3.5, are OK, 64-bit 3.6 is OK, but there's a test error for 32-bit Python 3.6:

```
======================================================================
ERROR: dipy.tracking.tests.test_utils.test_rmi
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\python36\lib\site-packages\nose\case.py", line 198, in runTest
    self.test(*self.arg)
  File "c:\python36\lib\site-packages\dipy\tracking\tests\test_utils.py", line 472, in test_rmi
    I2 = ravel_multi_index([A, B, C, D], dims=[1000] * 4)
ValueError: invalid dims: array size defined by dims is larger than the maximum possible size.
----------------------------------------------------------------------
```

https://ci.appveyor.com/project/matthew-brett/dipy-wheels/build/job/9u7643cl4xmif2r4#L3660

For Python 3.6 I had to use numpy 1.13, rather than 1.10.3 as for the other versions, because h5py requires it.  So that might be a part of the explanation.  Also the values in that test are random - but I got the same error running the tests locally.   Any thoughts?

**Comment by arokem** [comment]
Looks like a recurrence of https://github.com/nipy/dipy/issues/1299

**Comment by matthew-brett** [comment]
I thought I recognized it ...

**Comment by arokem** [comment]
I assume you are building this from the release version? Seems like this was resolved in #1300. Can we close this?

**Comment by matthew-brett** [comment]
Yes, right, I was building from 0.13.0.  Closing, thanks.
